README.md

```diff
@@ -110,6 +110,35 @@ If you are looking for the codebase of the current production version of GATK, p
   ./gatk-launch PrintReadsSpark -I input.bam -O output.bam
   ```
 
+####Running GATK4 with inputs on Google Cloud Storage:
+
+* GATK can read BAM or VCF inputs from a Google Cloud Storage bucket. Just use the "gs://" prefix:
+  ```
+  ./gatk-launch PrintReads -I gs://mybucket/path/to/my.bam -L 1:10000-20000 -O output.bam
+  ```
+* For public datasets, that will just work. For private datasets you must set up authentication.
```

Comment:
I was wrong. Technically the user doesn't need to be authenticated if you have an API key, but we aren't using one so the user needs to authenticate.